packages/vue-email/src/utils/markdown.ts

```diff
@@ -0,0 +1,337 @@
+import type { CSSProperties } from 'vue'
```

Comment:
I think this line should be above of `import type {  StylesType  } from '../types/markdown'` line